cli-commands/commands/test/specific/utils/index.js

```diff
@@ -1,8 +1,8 @@
 const assert = require('assert');
```

Comment:
Thanks for your PR @prcolaco , could you consider this refactoring, please. It is a little bit different: Message is a specific parameter for expectAssert only, so in my mind message checks should be done on expectAssert level instead of expectEOSError as the last one is an abstractions used by both expect functions
```
const assert = require('assert');

module.exports = {
    expectAssert: async function (promise, errorMessage) {
        const result = await expectEOSError(promise, 'eosio_assert_message_exception', 'assert');
        if (errorMessage && result.error) {
            result.error = result.message.search(errorMessage) >= 0;;
            result.text = `Expected assert with '${errorMessage}' message, got '${result.message}' instead`
        }

        assert(result.error, error.text);

    },
    expectMissingAuthority: async function (promise) {
        const result = await expectEOSError(promise, 'missing_auth_exception', 'missing authority');
        assert(result.error, result.text);
    }
}

const expectEOSError = async function (promise, errorType, errorInfo) {
    try {
        await promise;
    } catch (error) {
        const message = error.message || error;
        const expectedError = message.search(errorType) >= 0;
        return {
            message,
            error: expectedError,
            text: `Expected ${errorInfo}, got '${message}' instead`
        }
    }
    assert.fail(`Expected ${errorInfo} not received`);
}```

Reply:
I see your point. Done 👍